app/components/ui/atoms/FormHeadings.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { H2, H3 } from '@pubsweet/ui'
+import { css } from 'styled-components'
+
+const formSpacing = {
+  cssOverrides: {
+    H2: css`
+      margin-bottom: 36px;
+    `,
+    H3: css`
+      /* this reset from the theme doesn't work? works applied here */
+      margin-top: 0px;
+      margin-bottom: 24px;
+    `,
+  },
+}
+
+export const FormH2 = props => <H2 theme={formSpacing} {...props} />
+export const FormH3 = props => <H3 theme={formSpacing} {...props} />
```

Comment:
I would expect styles like this to be applied with styled components `extend` or `styled()` rather than a local css override.